### ISSUE TITLE
ADJOINT problem and RANS/NAVIER_STOKES solver does not work

### ISSUE THREAD

**Issue by CipollinoDuck** [su2code/SU2#1458]
### Discussed in https://github.com/su2code/SU2/discussions/1456

<div type='discussions-op-text'>

<sup>Originally posted by **CipollinoDuck** November 30, 2021</sup>
Hello everyone,

I ask here because I cannot find any solution elsewhere.

I am attempting an **Adjoint Optimization problem** with viscous solver (**RANS/NAVIER_STOKES**) using the **shape_optimization.py** script but I can't complete it because I receive the following message:

`Error in "void CSolver::Read_SU2_Restart_Binary(CGeometry*, const CConfig*, std::__cxx11::string)":`
`-------------------------------------------------------------------------`
`Unable to open SU2 restart file solution_adj_cd.dat`

With EULER solver it works fine, it creates the **solution_adj_*.dat** files everytime it runs the ADJOINT problem so, at the beginning, the process does not require any further file besides the .su2 and the .cfg ones.

Even the _Constrained shape design of a transonic turbulent airfoil tutorial_ does not work. The required files (_initial solution files for the solver and adjoint solution_flow.dat, solution_adj_cd.dat, and solution_adj_cmz.dat_) are given from the beginning and if they are not added to the path in which the tutorial will be run, it cannot be completed.

I also tried the **turb_ONERAM6 Test case** and personal files as well.

Furthermore, I tried to cheat (?) creating a solution_adj_cd.dat and solution_adj_cmz.dat files from the EULER ADJOINT problem because it was the only way I had to obtain that file: the RANS ADJOINT problem performed better than expected reaching a feasible geometry which is not the one obtained in the tutorial.

I do not get why I have this problem, if it is correct or not. Has anyone experienced this before?

Actually, I think I will run this kind of problem with the cheating (?) procedure described before because I need to go ahead with the work and it seems to work relatively fine.

In conclusion, the **ADJOINT** problem with **RANS/NAVIER_STOKES** solver does not work because it needs **solution_adj_*.dat** which are created autonomously with EULER solver.

Have I missed something written in other tutorials?

Thank you very much for all of the support.</div>

**Comment by pcarruscag** [comment]
Hi, please attach the exact files you are using, and commands you are running, to the discussion you already opened. That will make it easier for someone to help.
The best way to get help from a developer is to make it very simple to replicate the problem.
